features/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -27,6 +27,9 @@
     {% if console.app_base_name_for_history_api is defined %}
         "appBaseNameForHistoryAPI": "{{ console.app_base_name_for_history_api }}",
     {% endif %}
+    {% if console.allow_multiple_app_protocols is defined %}
```

Comment:
Can't we group this config under `console.ui`. WDYT?

Reply:
it is not only a UI change but also a behavioural change. IMO it should not group under console.ui